test/unit/fetchers/url_test.rb

```diff
@@ -88,6 +88,37 @@ def initialize(target, opts)
       _(res.resolved_source).must_equal({url: 'https://github.com/hardening-io/tests-os-hardening/archive/48bd4388ddffde68badd83aefa654e7af3231876.tar.gz',
                                          sha256: expected_shasum})
     end
+
+    %w{https://bitbucket.org/chef/inspec
+       https://bitbucket.org/chef/inspec.git
+       https://www.bitbucket.org/chef/inspec.git
+       http://bitbucket.org/chef/inspec
+       http://bitbucket.org/chef/inspec.git
+       http://www.bitbucket.org/chef/inspec.git}.each do |bitbucket|
+      it "resolves a bitbucket url #{bitbucket}" do
+           res = fetcher.resolve(bitbucket)
```

Comment:
Lines 99-104 are indented too far. I realize this is a slight nitpick, but I was initially confused where the enumerator loop ended (i.e. was the test starting at line 106 part of the loop and just outdented incorrectly?) and that shows how important indentation should be :)

99-102 should be indented 2 spaces in from the `it` in line 98, the `end` on 103 should line up with the `it` on 98, and the `end` on 104 should line up with the `%w` on 92.